tests/Makefile

```diff
@@ -1,29 +1,62 @@
-HAVE_AVX512 = $(shell grep avx512f /proc/cpuinfo)
-CXX_TARGETS = $(shell\
-  if [ -z "$(HAVE_AVX512)" ] ; then\
-    find . -maxdepth 1 -name "*_test.cc" ! -name "*512*";\
-  else\
-    find . -maxdepth 1 -name "*_test.cc";\
-  fi\
-)
-CXX_TARGETS := $(CXX_TARGETS:%.cc=%.x)
-
-CUDA_TARGETS = $(shell find . -maxdepth 1 -name "*cuda_test.cu")
-CUDA_TARGETS := $(CUDA_TARGETS:%cuda_test.cu=%cuda_test.x)
-
-CUSTATEVEC_TARGETS = $(shell find . -maxdepth 1 -name "*custatevec_test.cu")
-CUSTATEVEC_TARGETS := $(CUSTATEVEC_TARGETS:%custatevec_test.cu=%custatevec_test.x)
-
-HIP_TARGETS = $(shell find . -maxdepth 1 -name "*cuda_test.cu")
-HIP_TARGETS := $(HIP_TARGETS:%cuda_test.cu=%hip_test.x)
-
-GTEST_DIR = $(CURDIR)/googletest/googletest
-GMOCK_DIR = $(CURDIR)/googletest/googlemock
-
-CMAKE=cmake
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Determine the hardware features available in this CPU.
+HAVE_SSE := $(shell grep -qm 1 sse /proc/cpuinfo && echo "true")
```

Comment:
Nit: `-m 1` is unnecessary after `-q` which makes grep exit immediately after a match.